docs/Using-EthSigner/Using-EthSigner.md

```diff
@@ -0,0 +1,82 @@
+description: Using EthSigner
+<!--- END of page meta data -->
+
+# Using EthSigner
+
+EthSigner provides transaction signing and access to your keystore by implementing the following JSON-RPC 
+methods:   
+
+* [`eth_accounts`](#eth_accounts)
+
+* [`eth_sendTransaction`](#eth_sendtransaction)
+
+All other JSON-RPC requests are passed through EthSigner to Pantheon and the result from Pantheon returned 
+by EthSigner. 
+
+!!! note 
+    EthSigner supports the JSON-RPC service over HTTP only. 
+
+# eth_accounts
+
+Returns the account address with which EthSigner is signing transactions. That is, the account of the [keystore key file](../Using-EthSigner/Getting-Started.md#create-password-and-key-files).
```

Comment:
nit; 
`Returns the account address with which EthSigner is signing transactions.` -> 
`Returns the account address that EthSigner uses when signing transactions`

Reply:
Is this a matter of language or grammar or does it change the meaning ? I can't really see the meaning change if it's one. Can you explain the change ?

Reply:
Fixed